specification/communication/data-plane/Microsoft.CommunicationServicesPhoneNumbers/stable/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,963 @@
+          }
+        },
+        "capabilities": {
+          "$ref": "#/definitions/PhoneNumberCapabilities"
+        },
+        "areaCode": {
+          "description": "The area code of the desired phone number, e.g. 425.",
+          "type": "string"
+        },
+        "quantity": {
+          "format": "int32",
+          "description": "The quantity of desired phone numbers. The default value is 1.",
+          "default": 1,
+          "maximum": 2147483647,
+          "minimum": 1,
+          "type": "integer"
+        }
+      }
+    },
+    "PhoneNumberCost": {
+      "description": "The incurred cost for a single phone number.",
+      "required": [
+        "amount",
+        "billingFrequency",
+        "currencyCode"
+      ],
+      "type": "object",
+      "properties": {
+        "amount": {
+          "format": "double",
+          "description": "The cost amount.",
+          "type": "number"
+        },
+        "currencyCode": {
+          "description": "The ISO 4217 currency code for the cost amount, e.g. USD.",
+          "type": "string"
+        },
+        "billingFrequency": {
+          "description": "The frequency with which the cost gets billed.",
+          "enum": [
+            "monthly"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "BillingFrequency",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PhoneNumberSearchResult": {
+      "description": "The result of a phone number search operation.",
+      "required": [
+        "assignmentType",
+        "capabilities",
+        "phoneNumbers",
+        "phoneNumberType",
+        "searchId"
+      ],
+      "type": "object",
+      "properties": {
+        "searchId": {
+          "description": "The search id.",
+          "type": "string"
+        },
+        "phoneNumbers": {
+          "description": "The phone numbers that are available. Can be fewer than the desired search quantity.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "phoneNumberType": {
+          "description": "The phone number's type, e.g. geographic, or tollFree.",
+          "enum": [
+            "geographic",
+            "tollFree"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberType",
+            "modelAsString": true
+          }
+        },
+        "assignmentType": {
+          "description": "Phone number's assignment type.",
+          "enum": [
+            "user",
+            "application"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberAssignmentType",
+            "modelAsString": true
+          }
+        },
+        "capabilities": {
+          "$ref": "#/definitions/PhoneNumberCapabilities"
+        },
+        "cost": {
+          "$ref": "#/definitions/PhoneNumberCost"
+        },
+        "searchExpiresBy": {
```

Comment:
Should not `searchExpiresBy` be required as well?

Reply:
Yes. This was set to not required for a while since we weren't always getting this field populated in our test environment from TNM.

I'll mark this as required. And set "" as a default. But we should expect this field populated.

Also marked phone number cost as required.